graphql-dgs-graphiql-autoconfigure/src/main/java/com/netflix/graphql/dgs/graphiql/autoconfiguration/GraphiQLConfigurationProperties.kt

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Kotlin sources should be in `src/main/kotlin` (it's currently in the `java` dir)

Reply:
Good catch!

Reply:
Yeh those were already in there. I moved them in the subsequent PR and also made the packaging consistent `autoconfigure` / `autoconfiguration`.